OWNERS: Only request @infinisil for OWNERS changes

infinisil opened the pull request:
This was the original intention, but lower matching entries take precedence, so it never worked!

Note that before the move to ci/OWNERS, .github/CODEOWNERS had no owner itself, and I might also remove myself again if it turns out to be too much noise.

Ping @mweinelt 

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

philiptaron commented:
I'm interested in the spam, too. Could you keep me on?

philiptaron reviewed (dismissed):
Requesting changes to keep myself on.

infinisil committed:
OWNERS: Also make @philiptaron an owner of OWNERS

infinisil committed:
OWNERS: Only request @infinisil for OWNERS changes

This was the original intention, but lower matching entries take
precedence, so it never worked!

Note that before the move to ci/OWNERS, .github/CODEOWNERS had no owner
itself, and I might also remove myself again if it turns out to be too
much noise.
